domain/src/main/java/com/example/domain/seat/SeatState.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.domain.seat
+
+class SeatState(var isSelected: Boolean, val rank: SeatRank)
```

Comment:
```suggestion
data class SeatState(var isSelected: Boolean, val rank: SeatRank)
```
단순히 데이터를 wrapping 하는 목적이라면, data class를 활용하는 것을 권장합니다.

Reply:
```suggestion
class SeatState(val isSelected: Boolean, val rank: SeatRank)
```
var을 사용하게 되면, 외부에서 값을 변경할 수 있으며, 그렇게 되면 원본을 사용하고 있는 곳에서 예상치 못한 동작으로 이어지게 됩니다.
var 대신 val을 사용해보면 어떨까요?

Reply:
domain 으로 잘 위치시켰네요. 👍 
하지만, `isSelected` 라는 프로퍼티는 "선택" 여부에 대한 상태를 나타내며, "선택" 여부는 View와 관련이 있어 보입니다.
domain과는 거리가 멀어보이는데 어떻게 생각하시나요? 🤔

Reply:
리뷰어님 말씀대로 생각해보니 선택여부는 View의 영역인 것 같아 전처럼 TextView가 selected되었는지 여부를 파악해주는 식으로 로직을 수정해주었습니다!